tests/test_multipart.py

```diff
@@ -148,6 +162,22 @@ def test_read_chunk_requires_content_length(self):
         with self.assertRaises(AssertionError):
             yield from obj.read_chunk()
 
+    def test_read_chunk_properly_counts_read_bytes(self):
+        size = aiohttp.multipart.BodyPartReader.chunk_size * 10
+        obj = aiohttp.multipart.BodyPartReader(
+            self.boundary, {'CONTENT-LENGTH': size},
+            StreamWithShortenRead(b'.' * size + b'\r\n--:--'))
```

Comment:
@kxepal, not 8MB, but only 80KB (8192 * 10). Though, you are right. Even 2 bytes is enough.